far/FarEng.hlf.m4

```diff
@@ -6121,43 +6121,56 @@ files, like #Temporary panel# does.
-опции, выбранное вами в прошлый раз. Значение этой опции запоминается только до конца сеанса
-работы Far Manager. При следующем запуске Far опция снова будет установлена в зависимости от битов 0 и 1.
- 2. ^<wrap>Если бит 2 сброшен в 0, то опция "Права доступа" будет установлена в зависимости от
-битов 0 и 1 всякий раз при вызове диалога перемещения. Вы можете переключать эту опцию в диалоге
-вручную, но выбранное вами значение будет действовать только на текущую операцию переноса файлов.
-
- Аналогично, для операции копирования воздействие битов 3 и 4 зависит от состояния бита 5.
+ The #Access rights# option specifies the access rights assigned
+to newly created files and folders and can be one of:
 
- Примеры:
- 0x21 - ^<wrap>для операции перемещения опцию "Права доступа" выставлять всегда в "Копировать",
-для операции копирования выставлять опцию в значение по умолчанию и запоминать её значение до конца сеанса работы Far Manager.
- 0xС0 - ^<wrap>для перемещения запоминать значение опции до конца сеанса работы Far (при первом вызове диалога опция
-выставлена в значение по умолчанию), для операции копирования опцию "Права доступа" выставлять всегда в "Копировать".
+ #Default# - ^<wrap>Access rights are controlled by the file system;
+ #Copy#    - Copy access rights of the source objects;
+ #Inherit# - Inherit access rights of the parent folder.
+
+ The initial state of the #Access rights# option when the dialog
+is opened is controlled by three bits of the
+#System.CopySecurityOptions# parameter. Bits 0, 1, and 2 control the
+state of the option in the Move dialog; bits 3, 4, and 5 -- in the Copy
+dialog.
 
- По умолчанию значение параметра = 0 (опция "Права доступа" устанавливается в значение по
-умолчанию и до конца сеанса работы не запоминается).
+@=
+ Copy     Move     ^<wrap>Initial state of the #Access rights#
+ Dialog   Dialog   option when the dialog is opened
+@=
+ Bit 0    Bit 3    0 - ^<wrap>#Default# (bits 1 / 4 are ignored)
+                   1 – controlled by bits 1 / 4
 
- Примечание:
+ Bit 1    Bit 4    0 - ^<wrap>#Copy# (if bit 0 / 3 is set to 1)
+                   1 - #Inherit# (if bit 0 / 3 is set to 1)
 
- Параметр "System.CopySecurityOptions" не влияет на создание ссылок (Alt+F6). В этом случае
-права всегда копируются.
+ Bit 2    Bit 5    0 - ^<wrap>Defined by bits 0 и 1 / 3 и 4 of this parameter
+                   1 - The last user’s choice (within the current Far session)
 
- Изменение этого параметра возможно через ~far:config~@FarConfig@
+ Default value: 0 (when the dialog is opened, the #Access rights# option
+is always set to #Default#; user’s choices are not remembered).
+
+ Examples:
+
+ #0b100'001# - ^<wrap>In the Move dialog, the #Access rights# option
+is always set to #Copy#.
+             In the Copy dialog, the option is initially set to
+#Default#; then the previous user’s choice is remembered (within the
+current Far session).
+
+ #0b011'100# - ^<wrap>In the Move dialog, the #Access rights# option
```

Comment:
I like the binary notation, but it's not what the user will see in the config editor or an exported XML.
Maybe something like `011100 (28)`?

Reply:
See if you like how I put it.